prov/verbs/src/fi_verbs.h

```diff
@@ -613,6 +614,7 @@ struct vrb_ep {
 	struct rdma_conn_param		conn_param;
 	struct vrb_cm_data_hdr		*cm_hdr;
 	void				*cm_priv_data;
+	bool				hmem_enabled;
```

Comment:
This patch is combining at least 2 changes.  There's some lkey related change, and whatever this change is.  The patch description does not make it clear why the lkey change is being made.

Reply:
I am just going to add this optimization commit to the original HMEM support commit.